code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -8,6 +8,13 @@
 /datum/reagent/medicine
 	name = "Medicine"
 	id = "medicine"
+	overdose_threshold = 60
+
+/datum/reagent/medicine/overdose_process(mob/living/M)
+	if(prob(50))
+		M.adjustToxLoss(1*REM)
```

Comment:
dont forget to make charcoal have an OD too so that i can't just ignore 90% of med ODs with antitoxins like on bay